.github/workflows/main.yml

```
@@ -0,0 +1,23 @@
+name: Build and Deploy
+on: [push]
```

Can you change this to -

```suggestion
on:
  push:
    branches:
      - master
```

---

resolving this!